README.rst

```diff
@@ -59,7 +56,26 @@ To install in a virtual environment in your current project:
 Usage Example
 =============
 
-.. todo:: Add a quick, simple example. It and other examples should live in the examples folder and be included in docs/examples.rst.
+.. code-block:: python
+
+    import time
+    import board
+    import busio
+    import displayio
+    import adafruit_ssd1322
+
+    displayio.release_displays()
+
+    # This pinout works on a Metro and may need to be altered for other boards.
+    spi = busio.SPI(board.SCL, board.SDA)
```

Comment:
wanna use board.SCK and board.MOSI so its clearer its SPI not I2C?

Reply:
I prefer SCL and SDA because SCK and MOSI aren't labeled on the board.

Reply:
oki